frontend/src/app/components/excel-data/excel-data.component.ts

```diff
@@ -0,0 +1,66 @@
+import { Component, OnInit } from '@angular/core';
+import { ExcelData } from 'src/app/models/excel-data';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ibmpariwesh/socgen-stockmarket-project/pullRequest?prid=8405405)
```suggestion
import { ExcelData } from "src/app/models/excel-data";
```